sites/bizbash/server/templates/website-section/resources/maps-posters.marko

```diff
@@ -0,0 +1,44 @@
+import { getAsObject } from '@base-cms/object-path';
```

Comment:
This file should be deleted. It's a custom section page that doesn't apply to BizBash (I'm assuming it came from whatever site was copied).